src/stratigraphy/metadata/metadata.py

```
@@ -96,17 +87,101 @@
+
+        return cls(
+            elevation=elevation,
+            coordinates=coordinates,
+            language=language,
+            page_dimensions=page_dimensions,
+            filename=Path(filename),
+        )
+
 
 @dataclass
 class BoreholeMetadataList(metaclass=abc.ABCMeta):
```

Ruff

`BoreholeMetadataList` is an abstract base class, but it has no abstract methods

(B024)